.rubocop_todo.yml

```diff
@@ -0,0 +1,72 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2023-04-05 10:28:45 UTC using RuboCop version 1.48.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: CountComments, CountAsOne, AllowedMethods, AllowedPatterns.
+Metrics/MethodLength:
+  Max: 12
+
+# Offense count: 1
+# Configuration parameters: CountKeywordArgs, MaxOptionalParameters.
+Metrics/ParameterLists:
+  Max: 6
+
+# Offense count: 4
+# Configuration parameters: MinSize.
+Performance/CollectionLiteralInLoop:
+  Exclude:
+    - 'spec/defines/accounts_user_spec.rb'
+
+# Offense count: 4
+# Configuration parameters: Prefixes, AllowedPatterns.
+# Prefixes: when, with, without
+RSpec/ContextWording:
+  Exclude:
+    - 'spec/defines/accounts_user_spec.rb'
+
+# Offense count: 5
+# Configuration parameters: IgnoredMetadata.
+RSpec/DescribeClass:
+  Exclude:
+    - '**/spec/features/**/*'
+    - '**/spec/requests/**/*'
+    - '**/spec/routing/**/*'
+    - '**/spec/system/**/*'
+    - '**/spec/views/**/*'
+    - 'spec/acceptance/accounts_spec.rb'
+    - 'spec/acceptance/user_spec.rb'
+    - 'spec/defines/accounts_user_spec.rb'
+    - 'spec/functions/accounts_ssh_authorized_keys_line_parser_spec.rb'
+    - 'spec/functions/accounts_ssh_options_parser_spec.rb'
+
+# Offense count: 13
+# Configuration parameters: CountAsOne.
+RSpec/ExampleLength:
+  Max: 24
+
+# Offense count: 25
+RSpec/MultipleExpectations:
+  Max: 22
+
+# Offense count: 20
+# Configuration parameters: EnforcedStyle, IgnoreSharedExamples.
+# SupportedStyles: always, named_only
+RSpec/NamedSubject:
+  Exclude:
+    - 'spec/defines/accounts_user_spec.rb'
+    - 'spec/functions/accounts_ssh_authorized_keys_line_parser_spec.rb'
+
+# Offense count: 29
+# Configuration parameters: AllowedGroups.
+RSpec/NestedGroups:
+  Max: 5
+
+# Offense count: 1
+Style/MixinUsage:
```

Comment:
This seems to be a recurring cop violation in spec_helper.
I seen in other PRs this was disabled inline, do you think we should do the same here? Just curious

Reply:
Yeah, could do that.